app/views/user_mailer/password_changed.html.slim

```diff
@@ -3,10 +3,9 @@ html
     meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
   body
 
-    p You have asked Upaya to change the password currently associated with your Upaya Account.
+    p You have asked #{APP_NAME} to change the password currently associated with your #{APP_NAME} Account.
 
     p
-      'If you did not change your password, please contact the Upaya Customer Contact Center via the web form at
-      = link_to 'https://upaya.18f.gov/contact', 'https://upaya.18f.gov/contact'
+      'If you did not change your password, please contact the #{APP_NAME} Customer Contact Center via the web form at = link_to 'https://upaya.18f.gov/contact', 'https://upaya.18f.gov/contact'
```

Comment:
I'm not sure the `link_to` will actually be executed here. I think this will print the literal string `link_to`. Did you verify this by any chance? I'm not sure if we have a test for this. Also, this will make the line way over 100 characters. If you enable overcommit, it will warn you about slim errors and whether those errors are on lines you modified or not. Alternatively, you could run `slim-lint app/views` and see if any errors are on files you modified.